aep/general/0136/aep.md.j2

```diff
@@ -110,14 +103,14 @@ have no permanent effect on data within the API.
 - The URI **should** place both the verb and noun after the `:` separator
   (avoid a "faux collection key" in the URI in this case, as there is no
   collection). For example, `:translateText` is preferable to `text:translate`.
-- Stateless operations **must** use `POST` if they involve billing.
+- Stateless methods **must** use `POST` if they involve billing.
 
 ### Declarative-friendly resources
```

Comment:
This is guidance on the "management plane" resources really - although maybe we need to work on some nomenclature here.

I'd suggest either:

1. calling this "management plane resources" match the other AIPs (e.g. the create you're adding)
2. remove this guidance for now, and add in a follow-up PR once we get things ironed out.

thoughts?

Reply:
Discussed live:  Yusuke suggests just leaving it after all.